packages/amplify-provider-awscloudformation/lib/initializer.js

```diff
@@ -6,10 +6,11 @@ const ora = require('ora');
 const Cloudformation = require('../src/aws-utils/aws-cfn');
 const constants = require('./constants');
 const configurationManager = require('./configuration-manager');
+const { emitMetric } = require('./amplify-metric-emitter');
 
 function run(context) {
   return configurationManager.init(context)
-    .then((ctxt) => {
+    .then(async (ctxt) => {
```

Comment:
Why `async`? . We should either stick to promises all the way or to `async` and `await` everywhere in the same file.

Reply:
Will remove the async.